plugins/vendor/zoom/lib/model/processors/kZoomWebinarProcessor.php

```diff
@@ -39,9 +39,11 @@ protected function parseAdditionalUsers($additionalUsersZoomResponse)
 			$result = array();
 			foreach ($panelistsEmails as $panelistEmail)
 			{
+				$accessToken = kZoomOauth::getValidAccessToken($this->zoomIntegration);
```

Comment:
we need to take the user from zoom only on cms case, on other cases we will just have another unneeded call to zoom.
lets create a function on the processor parent that also check the user mode type like it was in the original function